### PR TITLE
Allow disabling of `xo/no-abusive-eslint-disable` via a comment

### LINKED ISSUE #40: Allow disabling of `xo/no-abusive-eslint-disable` via a comment
[Issuehunt badges](https://issuehunt.io/r/sindresorhus/eslint-plugin-unicorn/issues/40)

I don't think it's possible currently.

<details>
<summary>
<b>IssueHunt Summary</b>
</summary>

### Sponsors (Total: $40.00)

- [issuehunt issuehunt](https://issuehunt.io/u/issuehunt) ($40.00)

### Tips

- Checkout the [Issuehunt explorer](https://issuehunt.io/r/sindresorhus/eslint-plugin-unicorn/) to discover more funded issues.
- Need some help from other developers? [Add your repositories](https://issuehunt.io/r/new) on IssueHunt to raise funds.
</details>

### PR DESCRIPTION
**Pull request by TiagoDanin**
Closes #40

### CODE REVIEW

File: rules/no-abusive-eslint-disable.js [hunk 1]
```diff
@@ -14,15 +14,14 @@ const create = context => ({
 				!res[2] // But it did not specify any rules
 			) {
 				context.report({
-					// Can't set it at the given location as the warning
-					// will be ignored due to the disable comment
```
Comment (sindresorhus) [inline rules/no-abusive-eslint-disable.js]: Can you explain how the above does not apply anymore with your changes?

Comment (TiagoDanin) [reply rules/no-abusive-eslint-disable.js]: Eslint use the line and column position to determine whether a report will be displayed.
When the column is `-1` it is displayed (Will ignore the deactivation via `eslint-disable-line`, `eslint-disable-next-line` ...).

image
**Note**: `-1` becomes `0` in issue list

**Comment by sindresorhus** [comment]
Can you fix the merge conflict?

**Comment by TiagoDanin** [comment]
Done!

**Pull request merged**